### ISSUE TITLE
Error al descargar datos

### ISSUE THREAD

**Issue by angegazitua** [franciscogazitua/IIC2113-2024#8]
¡Hola! Luego de descargar los archivos y copiar la carpeta de datos y los json a Fire-Emblem y Fire-Emblem test solo me aparecen 14 test en vez de los 500 que salen en el video del profesor.... y cuando corro fire-emblem no me deja seleccionar correctamente el número de test. He descargado dos veces los archivo y no sé por qué ocurre esto :(

![image](https://github.com/franciscogazitua/IIC2113-2024/assets/80768594/b6c2f932-bde3-4a4d-92a7-a4a44b556f09)

![image](https://github.com/franciscogazitua/IIC2113-2024/assets/80768594/8806bc46-077b-47d9-9fe0-3d0982551524)

**Comment by cahinostroza** [comment]
Hola @angegazitua. Nos puedes mostrar el contenido de path_a_tu_solucion\Fire-Emblem\Fire-Emblem\bin\Debug\net8.0\data y path_a_tu_solucion\Fire-Emblem\Fire-Emblem.Tests\bin\Debug\net8.0\data? Además, puedes expandir el resultado de los test, para ver qué error muestra.

**Comment by angegazitua** [comment]
los tests me muestran lo siguiente:
![image](https://github.com/franciscogazitua/IIC2113-2024/assets/80768594/11b68c4a-6599-426f-b386-018d9b23e86a)

el contenido de fire emblem net.8.0/data es: 

![image](https://github.com/franciscogazitua/IIC2113-2024/assets/80768594/32be63e1-f67c-4520-85c9-41d86d04e612)

y el de fire-emblem test es: 

![image](https://github.com/franciscogazitua/IIC2113-2024/assets/80768594/caa6eae1-2e53-456c-abc8-47fe12a30a74)


¿Quizás es porque al descomprimir me queda como data/data y ahí los archivos?

**Comment by cahinostroza** [comment]
Efectivamente, es por eso, después de data deben ir altiro las carpetas de cada grupo de test

**Comment by angegazitua** [comment]
Perfecto, ahí lo logré. Muchas gracias :)
